OpenRA.Mods.Common/Effects/GravityBomb.cs

```diff
@@ -72,9 +74,10 @@ public void Tick(World world)
 			velocity -= acceleration;
 			pos += velocity;
 
-			if (pos.Z <= args.PassiveTarget.Z)
+			var dat = world.Map.DistanceAboveTerrain(pos);
+			if (dat.Length <= 0)
```

Comment:
So we are assuming `args.PassiveTarget.Z == 0`?
Wouldn't we want `if (dat.Length <= world.Map.DistanceAboveTerrain(args.PassiveTarget))` here?
E.g. what happens when we try to hit an air unit (with a lower cruise altitude) with an gravity bomb?